examples/tls/private/gen_certs.sh

```diff
@@ -0,0 +1,80 @@
+#! /bin/bash
+
+# Should use gen_ca.sh first to generate the CA.
```

Comment:
Why don't we simply generate it here?

Reply:
I separated the script for the case where the script user want to generate each certificate type separately. If the CA is generated each time, each CA generated will overwrite the previous CA. Should I add code that generates the CA only if it does not exist?

Reply:
I think so! We could add a "ca" case to the script that overwrites an existing CA cert (IE, does what the gen_ca.sh script does) and otherwise write out the CA cert if it doesn't exist.